website/content/en/v1.0/upgrading/v1-migration.md

```diff
@@ -626,6 +626,10 @@ Ensure these values are set for all resources **before** upgrading Karpenter.
 Failing to do so will result in Karpenter being unable to operate against those resources.
 For the AWS provider, the group will always be `karpenter.k8s.aws` and the kind will always be `EC2NodeClass`.
 
+### Forceful Expiration
+
+Expiration is now forceful and begins draining as soon as it’s expired without waiting for replacement capacity. Karpenter will start provisioning a replacement as soon as the node is expired and begins draining. This forceful approach means that all expired nodes hat were previously blocked from disruption (due to Pod Disruption Budgets (PDBs), do-not-disrupt pods, or other constraints) will be disrupted simultaneously. The scheduler will then attempt to provision new nodes for the disrupted workloads. This behavior may lead to an increased number of pods in the "Scheduling" state while replacement capacity is being provisioned. 
```

Comment:
I think there's a few points we want to emphasize with this:
- What was the previous state:
  - Expiration would not cause a node to begin draining until it could be done so **gracefully**. This means that there should be no pods scheduled that would block disruption, and Karpenter would provision a replacement node before beginning to drain the expired node.
- What is the current state:
  - Expiration is a **forceful** operation. Once a NodeClaim has expired, Karpenter will begin draining the Node immediately.
- What are the user implications:
  - Expiration no longer respects disruption budgets
  - Nodes with pods with blocking PDBs and `do-not-disrupt` annotations will be expired
  - After upgrading, any nodes which were expired (but where disruption was blocked) will be disrupted. This could result in a large number of Nodes disrupting simeltaneously.
    - It would be worth adding a recommendation to check for expired Nodes, and unblock expiration before upgrade to prevent cluster degradation.

Also, just a nit on doc wording, but we should try and be explicit about what resource we're referring to. It's easy to confuse resources without context, especially Nodes and NodeClaims. For example, looking at the first sentence:

>  Expiration is now forceful and begins draining as soon as it’s expired without waiting for replacement capacity.

It's not necessarily clear what's expiring, and what's being drained. I would error on the side of being explicit, even if it feels a little repetitive.

> NodeClaim expiration is now forceful. Once a NodeClaim expires, Karpenter will begin draining the corresponding Node immediatly, without first provisioning a replacement Node.